### ISSUE TITLE
end-of-line marker

### ISSUE THREAD

**Issue by thewizarodofoz** [perry-mitchell/ads.txt#4]
This package accepts only `\n` as an end-pf-line marker, but the [spec](https://iabtechlab.com/wp-content/uploads/2017/09/IABOpenRTB_Ads.txt_Public_Spec_V1-0-1.pdf) says:

> Individual records are separated by an end-of-line marker. The consumer systems should
liberally interpret CR, CRLF etc as a record separator.

I believe it is better to try to split the text into lines using several markers:
```js
const textSplitters = ['\n', '\r', '\r\n', '\n\r'];

function splitText(text) {
    for (let splitter of textSplitters) {
        const split = text.split(splitter);
        if (split.length > 1) return split;
    }
    return [text]; // or maybe throw an error
}
```

BTW - good job on this package and thanks for sharing!

**Comment by perry-mitchell** [comment]
Thanks!

This is a good idea, I'll add it now :)

**Comment by perry-mitchell** [comment]
Released in 0.3.0
